### PR TITLE
Adds `-h` usage message, as per ISSUE #231

### PR DESCRIPTION
**Pull request by schuylermartin45**
See #231

### CODE REVIEW

**Comment by schuylermartin45** [comment]
hmmmmm that's a good idea. Let me see, I bet there's a standard practice way of doing this.

**Comment by schuylermartin45** [comment]
Ask and you shall receive, my friend.

File: scripts/update_usage.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    data = None
+    with open(README) as file:
+        data = file.readlines()
+
+    begin_idx = find_line(data, BEGIN_SECTION)
+    end_idx = find_line(data, END_SECTION)
+
+    # Remove the old message and insert the new one in-place
+    data = data[: begin_idx + 1] + data[end_idx:]
+    data[begin_idx + 1 : begin_idx + 1] = usage_list
```
Comment (schuylermartin45) [inline scripts/update_usage.py:50]: This is a "insert list in place" trick, apparently.

File: Makefile [hunk 2]
```diff
@@ -66,7 +66,7 @@ environment:       ## Handles environment creation
 	conda run --name $(CONDA_ENV_NAME) pip install -e .
 
 pre-commit:        ## Runs pre-commit against files
-	pre-commit run --all-files
+	pre-commit run --verbose --all-files
```
Comment (schuylermartin45) [inline Makefile:69]: I personally find the `--verbose` mode significantly more useful than the default mode, but I can take it out if others disagree.

**Comment by schuylermartin45** [comment]
Looks like `pre-commit` failed on the import statement I use to get to `run.py` that works on my local.

**Comment by marcoesters** [comment]
> Looks like `pre-commit` as a GitHub Action failed on the import statement that I use to get to `run.py` that works on my local.

I suspect because you have `anaconda_linter` installed on your local machine, but it's not available in the pre-commit run. Maybe try and use a relative import?

**Comment by schuylermartin45** [comment]
oooh boy this is going to be a bit confusing. I thought it was pulling the `anaconda_linter` module that is adjacent to the `scripts` module as both have an `__init__.py`.

I'll try the relative path first. I was about to put a `__init__.py` in the top-level to see if it needed that to get the adjacent module to identify correctly.

**Comment by schuylermartin45** [comment]
I'm going to close this PR and put up another shortly that handles the manual inclusion of the help message.

Mostly because I want to reference this PR as-is in a few new issues.

**Pull request closed without merging**